vfio_user/Cargo.toml

```diff
@@ -13,4 +17,9 @@ serde_json = "1.0.93"
 thiserror = "1.0.38"
 vfio-bindings = { git = "https://github.com/rust-vmm/vfio", branch = "main", features = ["fam-wrappers"] }
 vm-memory = { version = "0.10.0", features = ["backend-mmap", "backend-atomic"] }
```

Comment:
To build the gpio example (with `cargo build -p vfio_user --example gpio`, I will need to enable the `"backend-bitmap"` feature for the `vm-memory` crate, otherwise I will get the following error

```
error[E0432]: unresolved import `vm_memory::bitmap::AtomicBitmap`                                                                                                                                            
  --> pci/src/vfio_user.rs:23:5                                                                                                                                                                               
   |                                                                                                                                                                                                          
23 | use vm_memory::bitmap::AtomicBitmap;                                                                                                                                                                     
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `AtomicBitmap` in `bitmap`                                                                                                                                        
                                                                                                                                                                                                              
For more information about this error, try `rustc --explain E0432`
```

Reply:
Okay, so I didn't see that because I used `cargo build --all --examples` to build it. The correct fix was to ensure the pci crate used the correct features for the vm-memory crate.

Reply:
I did enhance the GH workflow to ensure buildability!